### DISCUSSION TITLE
keymap <c-n> and <c-p> to move up and down when you are on seach

### DISCUSSION THREAD

**Discussion by simanga-dev** [jarun/nnn#1839] [General]
Hello everyone, following Vim, it is a common keymap that allows you to move up
by pressing control-p and down by pressing control-n. I would love to have this
feature in NNN as well. I can contribute to it if it is something that can be
merged.

**Comment by N-R-K** [comment]
Both ctrl-n and ctrl-p are already taken.

**Reply by simanga-dev** [reply]
oOh, I forget to mention, I meant in search particular.... when you type `/` type the words.

**Reply by N-R-K** [reply]
It's still taken because any binding starting with `ctrl` is also active in search mode. You can change the binding for your personal build if you want, see `nnn.h` file: https://github.com/jarun/nnn/blob/master/src/nnn.h
